CHANGELOG.md

```diff
@@ -2,12 +2,13 @@
 
 ## Unreleased
 
+## 15.3.0
+- Add rust_server feature to glean_parser to allow Rust server-side metrics ([#772](https://github.com/mozilla/glean_parser/pull/772))
+
 ## 15.2.1
 
 - Allow earlier versions of platformdirs ([#769](https://github.com/mozilla/glean_parser/pull/769))
 
-## 15.2.0
-
```

Comment:
huh, this should definitely not go away.

Reply:
Good catch, I think when I rebased it removed that line for some reason and I didn't see. Thx